earth_enterprise/src/fusion/autoingest/StorageManager.h

```diff
@@ -38,9 +43,55 @@ class StorageManaged {
     StorageManaged() : timestamp(0), filesize(0) {}
 };
 
+template<class AssetType, class AssetStorageType> class AssetHandleInterface;
```

Comment:
Minor: I don't think this line is necessary. It must have been pulled in as part of the merge.

Reply:
Good catch. Removed it.